Datadog/E2ETests/RUM/RUMMonitorE2ETests.swift

```diff
@@ -0,0 +1,679 @@
+        }
+        Thread.sleep(forTimeInterval: RUMConstants.actionInactivityThreshold)
+        rum.sendRandomActionOutcomeEvent()
+    }
+
+    /// - data monitor:
+    /// ```rum
+    /// $monitor_id = rum_rummonitor_start_resource
+    /// $monitor_name = "[RUM] [iOS] Nightly - rum_rummonitor_start_resource: number of views is below expected value"
+    /// $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_start_resource @view.url_path:datadog\/rum* @type:resource\").rollup(\"count\").last(\"1d\") > 1"
+    /// ```
+    func test_rum_rummonitor_start_resource() {
+        let viewKey = String.mockRandom()
+        let viewName = String.mockRandom()
+        let resourceKey = String.mockRandom()
+
+        rum.startView(key: viewKey, name: viewName, attributes: DD.logAttributes())
+
+        measure(resourceName: DD.PerfSpanName.fromCurrentMethodName()) {
+            rum.startResourceLoading(
+                resourceKey: resourceKey,
+                httpMethod: .get,
+                urlString: String.mockRandom(),
+                attributes: DD.logAttributes()
+            )
+        }
+
+        rum.stopView(key: viewKey, attributes: [:])
+    }
+
+    /// - data monitor:
+    /// ```rum
+    /// $monitor_id = rum_rummonitor_stop_resource
+    /// $monitor_name = "[RUM] [iOS] Nightly - rum_rummonitor_stop_resource: number of views is below expected value"
+    /// $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_stop_resource @view.url_path:datadog\/rum* @type:resource @resource.status_code:200 @resource.type:(beacon OR fetch OR xhr OR document OR unknown OR image OR js OR font OR css OR media OR other) @resource.url:http\:\/\/datadog\/resource\/rum\/*\").rollup(\"count\").last(\"1d\") < 1"
+    /// ```
+    func test_rum_rummonitor_stop_resource() {
+        let viewKey = String.mockRandom()
+        let viewName = String.mockRandom()
+        let resourceKey = String.mockRandom()
+
+        rum.startView(key: viewKey, name: viewName, attributes: DD.logAttributes())
+
+        rum.startResourceLoading(
+            resourceKey: resourceKey,
+            httpMethod: .get,
+            urlString: String.mockRandom(),
+            attributes: DD.logAttributes()
+        )
+        measure(resourceName: DD.PerfSpanName.fromCurrentMethodName()) {
+            rum.stopResourceLoading(resourceKey: resourceKey, statusCode: 200, kind: .other)
+        }
+
+        rum.stopView(key: viewKey, attributes: [:])
+    }
+
+    // TODO: RUMM-1378  query below can't be parsed:
+    // $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_stop_resource_with_error @type:error @error.resource.status_code:>=400 @error.source:(logger OR network OR source OR console OR agent OR webview) @error.resource.url:http\:\/\/datadog\/resource\/rum*\").rollup(\"count\").last(\"1d\") < 1"
```

Comment:
with this line, linter gives `$monitor_query` is not defined error
i think somehow it can't parse this line, the problem disappears when i remove `@error.resource.status_code:>=400` so `>=` should be the root cause